test/renderer/plugins/cuaSettings.test.ts

```
@@ -0,0 +1,77 @@
+      }),
+      invokeAction: vi.fn().mockResolvedValue({
+        ok: true,
+        data: {
+          accessibility: 'granted',
+          screenRecording: 'denied'
+        }
+      }),
+      disable: vi.fn()
+    }
+
+    window.eval(await readFile(scriptPath, 'utf8'))
```

Biome (2.4.14)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)